### ISSUE TITLE
loss NAN or INF

### ISSUE THREAD

**Issue by WannaFuture** [yanghangAI/PRIS#2]

**Comment by Kirby13740** [comment]
I also get the training loss NAN, have you solved the problem now?

**Comment by WannaFuture** [comment]
> I also get the training loss NAN, have you solved the problem now?

I've been researching this issue for a while now, and here are some measures that I think will mitigate loss NAN:
1. for convolution operations in invblock, use weighted initialization (e.g. Xavier)
2. use a smaller learning rate (1e-5 recommended)
3. check for bugs in the code
By the way, flow-based models are really not very stable and it is better to use off-the-shelf frameworks.
Hope these above can help you.
